scripts/ci-publish.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -e # Exit with nonzero exit code if anything fails
+
+###############################################################################
+# Config NPM
+###############################################################################
```

Comment:
Do we need these banners? I don't really like them here.

Reply:
The script was initially bigger and made more sense. Still, bash scripts can get unwieldy. I like setting a standard that can carry forward if/when this scripts evolves.